pkg/provisioner/ironic/testbmc/testbmc.go

```diff
@@ -75,7 +75,7 @@ func (a *testAccessDetails) PowerInterface() string {
 }
 
 func (a *testAccessDetails) RAIDInterface() string {
-	return ""
+	return "no-raid"
```

Comment:
What about using a constant for such value?

Reply:
we can probably think about creating possible values for raid as a constant (In a different PR I would say, since this is blocking deployments with virtual media)